composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "seblucas/php-epub-meta",
+    "type": "library",
+    "description": "Reading and writing metadata included in the EPub ebook format",
+    "keywords": ["epub", "metadata", "ebook"],
+    "homepage": "https://github.com/seblucas/php-epub-meta",
+    "authors": [
+        {
+            "name": "Andreas Gohr",
+            "email": "andi@splitbrain.org",
+            "homepage": "https://www.splitbrain.org/",
+            "role": "Developer"
+        },
+        {
+            "name": "Sébastien Lucas",
+            "email": "sebastien@slucas.fr",
+            "homepage": "http://www.slucas.fr/",
+            "role": "Developer"
+        }
+    ],
+    "require": {
+        "php": ">=5.3.0",
+        "ext-xml": "*",
+        "ext-zip": "*",
+        "seblucas/tbszip": "~2.16.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "5.4.*"
```

Comment:
I guess this is not compatible with php 5.3.

Any version should enough for the tests, I think